README.md

```diff
@@ -4,7 +4,89 @@ OpenModelica Windows installer based on [NSIS](https://nsis.sourceforge.io/Main_
 
 ## Dependencies
 
-- [NSIS](https://nsis.sourceforge.io/Main_Page)
+  - [NSIS 3.0.4](https://nsis.sourceforge.io/Main_Page)
+    - Copy `AccessControlW.dll` into your NSIS plugin directory, e.g. into
+      `C:\Program Files (x86)\NSIS\Plugins\x86-unicode`.
+  - [git](https://git-scm.com/)
+  - [OMDev](https://gitlab.liu.se/OpenModelica/OMDevUCRT)
+  - [SignTool](https://learn.microsoft.com/en-us/windows/win32/seccrypto/signtool),
```

Comment:
`SignTool` is actually not needed to build the installer. It is more like a post process thing that we do before uploading the installer.

Reply:
Yes, I skipped that step as well when building the installer locally.
But I think we should keep it in README.

Reply:
Sure, but then just mark these steps as optional.